internal/internal_event_handlers.go

```diff
@@ -494,11 +494,12 @@ func (wc *workflowEnvironmentImpl) ExecuteLocalActivity(params executeLocalActiv
 
 func newLocalActivityTask(params executeLocalActivityParams, callback laResultHandler, activityID string) *localActivityTask {
 	task := &localActivityTask{
-		activityID:  activityID,
-		params:      &params,
-		callback:    callback,
-		retryPolicy: params.RetryPolicy,
-		attempt:     params.Attempt,
+		activityID:   activityID,
+		params:       &params,
+		callback:     callback,
+		retryPolicy:  params.RetryPolicy,
+		attempt:      params.Attempt,
+		workflowTask: &workflowTask{},
```

Comment:
why is this needed ? localActivityTask shouldn't and doesn't need a non-nil workflowTask